acceptance/tests/options/config_file/debug.rb

```diff
@@ -22,8 +24,8 @@
       end
 
       step "debug output should print when config file is loaded" do
-        on(agent, facter("")) do
-          assert_match(/DEBUG/, stderr, "Expected DEBUG information in stderr")
+        on(agent, facter("")) do |facter_output|
```

Comment:
John said out that I should be doing it this way so there is not a confusion of where the stdout or stderr is coming from.